### ISSUE TITLE
Drop DataFrames dependency?

### ISSUE THREAD

**Issue by hustf** [scheinerman/LatexPrint.jl#9]
Let me apologize for making such a suggestion without having the resources to follow up with a PR. 

I am just looking for a set of packages to support experimenting with an equation editor which updates .jl files (probably being way too ambitious). I would need to do a lot of what you already do here, but the DataFrames dependency is a heavy one, when all the indirect dependencies are included. 

For my own simple uses, I found IndexedTables to be a useable replacement.

**Comment by scheinerman** [comment]
I don't use `DataTables` myself, so I don't recall why I have it in there at all. I'll need to take a look and see what the consequences might be in pulling it out.

But I'm not sure why the inclusion is a problem or its removal would solve your problem. I guess I really don't understand what you're trying to do.

**Comment by hustf** [comment]
Thank you for the explanation. If I find this package is the best fit otherwise, I will make a pr of it.

If you are interested in the browser based equation editor for julia input and output, I can share the work in progress with you. My vision is something similar to the Mathcad and smath gui.  I started in a private repository because I did not have the complete picture of involved license types. It will be both apache and mit license it seems, because mathjax uses apache license. I would prefer something like mathjax that is using ES6 javascript.
